roles/kube-gateway/templates/gateway.yml.j2

```diff
@@ -32,6 +32,10 @@ spec:
         value: "{{ datadog_api_key }}"
       - name: TAGS
         value: "KubernetesCluster:{{ cluster }},KubernetesClusterRole:{{ cluster_role }}"
+      - name: STATSD_HOST
+        value: {{ datadog_statsd_host|default('') }}
```

Comment:
How about we set the same defaults that are indicated in the config, just in case the underlying code needs to change for any reason in the future